### PR TITLE
change file

### PR DESCRIPTION
**Pull request by tpauling**
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tpauling/testninja/pull/1%23issuecomment-129438136%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-08-10T13%3A10%3A53Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/13135987%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tpauling%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%3Agun%3A%22%2C%20%22created_at%22%3A%20%222015-08-11T08%3A56%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/13135987%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tpauling%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-08-11T08%3A56%3A18Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/13135987%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tpauling%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/tpauling/testninja/pull/1%23issuecomment-129438136%22%2C%20%22https%3A//github.com/tpauling/testninja/pull/1%23issuecomment-129777087%22%2C%20%22https%3A//github.com/tpauling/testninja/pull/1%23issuecomment-129777262%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/tpauling/testninja/pull/1#issuecomment-129438136'>General Comment</a></b>
- <a href='https://github.com/tpauling'></a> :shipit:
- <a href='https://github.com/tpauling'></a> :shipit::gun:

<a href='https://www.codereviewhub.com/tpauling/testninja/pull/1?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/tpauling/testninja/pull/1?approve=1'></a>&nbsp;<a href='https://github.com/tpauling/testninja/pull/1'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -3,3 +3,5 @@ Hello World!
 THIS IS SPARTA!
 
 TEST TEST 123 TEST TEST 123
+
+BEEDOOOOO BEDDDOOOOOOOOOOO!!!!!!!!!!!
```
Comment (tpauling) [inline file.txt]: Minions!!!! ;)

File: file.txt [hunk 2]
```diff
@@ -3,3 +3,5 @@ Hello World!
 THIS IS SPARTA!
```
Comment (tpauling) [inline file.txt]: 300

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/tpauling/testninja/pull/1" target="_blank">ReviewNinja

File: file.txt [hunk 3]
```diff
@@ -3,3 +3,5 @@ Hello World!
...
 
 TEST TEST 123 TEST TEST 123
+
```
Comment (tpauling) [inline file.txt]: add a new movie wisdom :) !fix

File: file.txt (hunk 1 above)
Comment (tpauling) [inline file.txt]: !completed

Comment (tpauling) [inline file.txt]: !fixed

Comment (tpauling) [inline file.txt]: !completed

**Comment by tpauling** [comment]
:shipit:

**Comment by tpauling** [comment]
:shipit::gun:

File: file.txt (hunk 1 above)
Comment (tpauling) [inline file.txt]: :+1:<a href='#crh-update-complete'></a>

**Comment by tpauling** [comment]
:+1:<a href='#crh-update-none'></a>

File: file.txt (hunk 1 above)
Comment (tpauling) [inline file.txt]: :+1:

**Comment by tpauling** [comment]
:shipit:

**Comment by tpauling** [comment]
:shipit::gun:

**Comment by tpauling** [comment]
:+1:

**Pull request closed without merging**